scene_id: 0
main_loop_freq: 50.0
unity_render: yes 

t_BC_x: 0.0
t_BC_y: 0.0
t_BC_z: 0.3

q_BC_w: 1.0
q_BC_x: 0.0
q_BC_y: 0.0
q_BC_z: 0.0

use_rgb: true
use_depth: true
use_semantics: true
use_optical_flow: false
